### ISSUE TITLE
Not editable entity fields are zeroed/nullified when entity is edited in web admin

### ISSUE THREAD

**Issue by servy** [la-team/light-admin#233]
If an entity has a persistable field that should not be changed via admin interface (so the field is hidden in formView) that field is zeroed after editing the entity. Sample entity:

```java
@Entity
@Table(name = "test_items")
public class TestItem {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "text")
    private String text;

    @Column(name = "counter")
    private int counter = 42;
    
    // getters and setters omitted for brevity
}
```

Sample administration configuration:

```java
public class TestItemAdministration extends AdministrationConfiguration<TestItem> {
    @Override
    public FieldSetConfigurationUnit formView(PersistentFieldSetConfigurationUnitBuilder fragmentBuilder) {
        // counter field is not editable via admin interface
        return fragmentBuilder
                .field("text").editor(Editors.wysiwyg())
                .build();
    }
}
```

Creating an entity works as expected with counter field value of 42. Editing the entity (changing text) and saving leads to counter field being set to zero. Expected behavior for the counter field is to preserve its value after editing the entity.

Same problem exists for object types (expirienced it with DateTime). Object fields are set to null after editing an entity if those fields are not shown in the formView. If database has NOT NULL constraint on the field saving changes fails with an exception.

First revision affected by this issue is a9d44efa55b2c146ae4a6c2f55fbb4db3f2cb718. Earlier revisions work fine.

**Comment by agustisanchez** [comment]
Yes, this is a big bug as all my entities have a `creationDate` field that I don't want to show in the form view and which should not be set to null (or overridden with any value) during update operations.

Currently, all update operations fail in my app because of that.

**Comment by agustisanchez** [comment]
Related to https://github.com/la-team/light-admin/issues/150 .

Apparently it was fixed in 1.1.0.RELEASE. But it's still there in 1.0.1.RELEASE and 1.2.0.RC1.

**Comment by anegrin** [comment]
it does not look like a light-admin issue but a spring-data/jackson issue.

i solved this by annotating my entities and/or mapped superclass with

`@JsonIgnoreProperties({"createdAt", "createdBy", "updatedAt", "updatedBy"})`

where "createdAt", "createdBy", "updatedAt", "updatedBy" are my auditing properties, no present in the form but stored in the db

**Comment by schloemi** [comment]
@JsonIgnoreProperties({"createdAt", "createdBy", "updatedAt", "updatedBy"}) is not really an option, cause fields will also not available in listView oder quickView.

Will this be fixed in future releases?

**Comment by Intelligent2013** [comment]
I found that this issue was fixed the version 1.1.0.BUILD-SNAPSHOT 20140917.172357-18. But the version 20150530.133354-43 has this issue again. I would like to use more newer version than 20140917. Could you inform me, in what last version (i.e. before 20150530.133354-43) the issue still fixed?
